docs/Getting-Started/Private-Network-Quickstart.md

```diff
@@ -164,7 +182,6 @@ The result should be as follows:
 ```
 Here we simply query the version of the Pantheon node, which confirms the node is running.
 
-Now if this works, let's see some more interesting requests.
```

Comment:
I liked the idea to tell that if it doesn't work here, no need to go further. How could we tell that differently ?

Reply:
How about this: "This shows that you can connect to the nodes via RPC. The rest of this section demonstrates more interesting requests. You can walk through them or skip ahead to "Creating a Transaction Using MetaMask."